br/pkg/backup/client.go

```diff
@@ -1270,12 +1297,17 @@ func (bc *Client) handleFineGrained(
 	storeID := targetPeer.GetStoreId()
 	lockResolver := bc.mgr.GetLockResolver()
 	client, err := bc.mgr.GetBackupClient(ctx, storeID)
+
 	if err != nil {
 		if berrors.Is(err, berrors.ErrFailedToConnect) {
 			// When the leader store is died,
 			// 20s for the default max duration before the raft election timer fires.
 			logutil.CL(ctx).Warn("failed to connect to store, skipping", logutil.ShortError(err), zap.Uint64("storeID", storeID))
-			return 20000, nil
+			return 20000, &StoreBasedErr{
+				storeID: storeID,
+				message: "failed to connect to store",
```

Comment:
does it need message? looks like it's always using the same message

Reply:
Done